packages/printbox/printbox.v0.3/opam

```diff
@@ -0,0 +1,35 @@
+opam-version: "2.0"
+version: "0.3"
```

Comment:
```suggestion
```
This field is unnecessary and doesn't match the package version. Could you move the package from `printbox.v0.3` to `printbox.0.3`?

Reply:
why does it matter what the git tag is? I'd rather not, really…

Reply:
The version of the path is the one used by opam, it overrides the one in the opam file and shows a warning. The path should use the correct version, in this case it seems 0.3